### DISCUSSION TITLE
Guest license

### DISCUSSION THREAD

**Discussion by RickCogley** [outline/outline#1778] [Self Hosting] [answered]
Hi - I searched but did not find anything about this: 
Does a Guest user use up a paid license slot?

**Comment by matbrgz** [comment]
Yes, at this moment Outline does not support guest.

**Comment by tommoor** [comment, marked as answer]
> Does a Guest user use up a paid license slot?

They take up a full license, yes. A guest user has different sign-in options in the form of email but other than that they have identical functionality to other team members.
